spec/features/service_drafts/preview/empty/service_information_preview_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
```

Comment:
is this one of the tests that got 'lost'?

Reply:
yep,

`service_drafts/preview/*/service_information_preview_spec.rb` and
`service_drafts/preview/*/service_keywords_preview_spec.rb`